azurerm/resource_arm_cosmos_db_account.go

```diff
@@ -198,6 +198,7 @@ func resourceArmCosmosDBAccount() *schema.Resource {
 								"EnableTable",
 								"EnableGremlin",
 								`EnableCassandra`,
+								"EnableAggregationPipeline",
```

Comment:
can we add this value to [the list of acceptable values in the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/cosmosdb_account.html.markdown#L78) too?

Reply:
Done :)